package.json

```diff
@@ -39,9 +39,9 @@
     "@mui/icons-material": "^5.8.4",
     "@mui/lab": "^5.0.0-alpha.72",
     "@mui/material": "^5.5.0",
-    "@pushprotocol/restapi": "1.6.12-alpha.5",
+    "@pushprotocol/restapi": "1.7.15",
     "@pushprotocol/socket": "0.5.3",
-    "@pushprotocol/uiweb": "1.2.11",
+    "@pushprotocol/uiweb": "1.3.1",
```

Comment:
The pull request heading says using uiweb alpha and the version is changed here. What's the correct expected version here?

Reply:
Was publishing alpha but changes are made to stable release as well so switched to the latest version